src/yunohost/app.py

```diff
@@ -2506,10 +2506,10 @@ def unstable_apps():
 
     for infos in app_list(full=True)["apps"]:
 
-        if not infos.get("from_catalog") or infos.get("from_catalog").get("state") in [
+        if not infos.get("from_catalog") or infos.get("from_catalog", {}).get("state") in [
             "inprogress",
             "notworking",
-        ] or infos["id"] in deprecated_apps:
+        ] or infos["id"] in deprecated_apps or infos.get("from_catalog", {}).get("level") == 0:
```

Comment:
Ugh yeah but sometimes some apps just get level 0 temporarily on the CI but are still actively maintained and alive, we need to differentiate those two ... In this code, as far as I remember I assumed that actually non-maintained app would get flagged as "notworking" in the catalog. The code implicitly check this because only working (and maybe inprogress? not sure) apps are listed in the actual "built" catalog.

Why not just add `ffsync` in the `deprecated_apps` list just a few lines before ?

Reply:
This `unstable_apps` function is used only by buster and bullseye migration. So if an app is level  in bullseye it could be a good idea to discourage migrating until the issues has been fix (in the case the person had to reinstall or force upgrade the app).